### DISCUSSION TITLE
[FR] - Add a button to go to the top of the list

### DISCUSSION THREAD

**Discussion by bepolymathe** [TechbeeAT/jtxBoard#880] [Ideas]
Hi @patrickunterwegs 

I have a fairly large number of tasks and I'd find it interesting and practical to have a button to go to the top of the list... 

![image](https://github.com/TechbeeAT/jtxBoard/assets/29939805/9a118c2f-5ec4-4835-9b82-c15eba3057ae)

Maybe to the right of three dots ?

**Comment by patrickunterwegs** [comment]
Hey @bepolymathe , thank you, that's already covered here: #860 😉

I'm thinking more about a floating button that appears when you are not on top of the list!

**Reply by bepolymathe** [reply]
Oh tha’s great ! Thanks

**Comment by bepolymathe** [comment]
Close by #860
